src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -79,6 +82,85 @@ public function getLocator()
      */
     public function import($resource, $type = null, $ignoreErrors = false, $sourceResource = null)
     {
+        $ret = array();
+        $ct = 0;
+        foreach ($this->glob($resource, false, $_, $ignoreErrors) as $resource => $info) {
+            ++$ct;
+            $ret[] = $this->doImport($resource, $type, $ignoreErrors, $sourceResource);
+        }
+
+        return $ct > 1 ? $ret : $ret[0] ?? null;
```

Comment:
even if already ugly enough as is, do we really want this to require php 7+ for such syntactic sugar `??`?